Compiler/SimCode/SimCodeUtil.mo

```diff
@@ -279,6 +281,10 @@ protected
   BackendDAE.Variables setcVars;
   list<SimCodeVar.SimVar> tmpsetcVars;
   SimCode.JacobianMatrix dataReconSimJac;
+  String fullPathPrefix;
+
+  SimCode.OMSIFunction omsiAllEquations, omsiInitEquations;
```

Comment:
The naming is a bit confusing. As far as I understand, `omsiAllEquations` refers to the ode used for continuous simulation, right?

Reply:
Well it has all equations needed to simulate after initialization, including discrete ones. 
Maybe `omsiSimEquations` would be clearer and consistent to the definition in `OMSIData`.